### PR TITLE
[ACS][CallAutomation] Adding TeamsAppidentifier + Lint fixing

### PR DESCRIPTION
**Pull request by minwoolee-msft**
Adding TeamsAppidentifier + Lint fixing

### CODE REVIEW

File: sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_connection_client_async.py [hunk 1]
```diff
@@ -1010,15 +1011,15 @@ async def hold(
         *,
         play_source: Optional[Union['FileSource', 'TextSource', 'SsmlSource']] = None,
         operation_context: Optional[str] = None,
-        operation_callback_uri: Optional[str] = None,
+        operation_callback_url: Optional[str] = None,
```
Comment (l0lawrence) [inline sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_connection_client_async.py:1014]: why was this renamed? Wouldn't this be a breaking change @kashifkhan?

File: sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py [hunk 2]
```diff
@@ -711,7 +714,8 @@ def start_continuous_dtmf_recognition(
         """
         continuous_dtmf_recognition_request = ContinuousDtmfRecognitionRequest(
             target_participant=serialize_identifier(target_participant),
-            operation_context=operation_context
+            operation_context=operation_context,
+            operation_callback_uri=operation_callback_url
```
Comment (l0lawrence) [inline sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py:718]: called _uri here?

**Review by l0lawrence — COMMENTED** [review]
thanks! Can we remove the other changes and make a just linting pr?

File: sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_connection_client_async.py (hunk 1 above)
Comment (minwoolee-msft) [reply sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_connection_client_async.py:1014]: This is mistake - it should be ..._url, like other instances.
CallAutomation has branching policy where we merge master with latest features ("alpha features") and we have separate branch for release. It should be ok to make changes here, as this only changes in alpha

File: sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py (hunk 2 above)
Comment (minwoolee-msft) [reply sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py:718]: it is ..._uri in request (autorest generated) so this is correct

**Review by l0lawrence — APPROVED** [review]
